src/sentry/integrations/vercel/integration.py

```diff
@@ -212,16 +212,6 @@ def update_organization_config(self, data):
             sentry_project_dsn = enabled_dsn.get_dsn(public=True)
 
             vercel_project = vercel_client.get_project(vercel_project_id)
-            source_code_provider = vercel_project.get("link", {}).get("type")
```

Comment:
@lforst We will lose releases if the user doesn't have a source code provider. And I think other things might be messed up as well. When is this situation happening?